tests/unit/generators/schema/breadcrumb-test.php

```diff
@@ -685,41 +685,11 @@ public function test_is_not_needed_on_error_page() {
 	}
 
 	/**
-	 * Tests that breadcrumbs are not shown on the home page.
+	 * Tests that breadcrumbs are shown in all other cases.
 	 *
 	 * @covers ::is_needed
 	 */
-	public function test_is_not_needed_on_home_page() {
-		$this->meta_tags_context->indexable->object_type = 'home-page';
-		$this->assertFalse( $this->instance->is_needed() );
-	}
-
-	/**
-	 * Tests that breadcrumbs are not shown on static home pages.
-	 *
-	 * @covers ::is_needed
-	 */
-	public function test_is_not_needed_on_static_home_page() {
```

Comment:
Why is there no longer a test for the static homepage? We still want no breadcrumbs on static homepages, right?